src/main/java/com/slimebot/alerts/holidays/HolidayAlert.java

```diff
@@ -82,21 +86,31 @@ private void sendMessage(JsonObject object) {
 
 
 	// date = yyyy-MM-dd
-	private JsonObject getObjectAtDate(String date) throws URISyntaxException, IOException, ParseException {
+	private List<JsonObject> getObjectsAtDate(String date) throws URISyntaxException, IOException, ParseException {
 		HttpGet request = new HttpGet(apiUrl.toURI());
 
-		CloseableHttpResponse response = httpClient.execute(request);
-		HttpEntity entity = response.getEntity();
+		AtomicReference<JsonArray> atomicArray = new AtomicReference<>(new JsonArray());
```

Comment:
I feel like it would be cleaner to make the method return void and instead give a Consumer<JsonArray> as parameter to that method that handle the response

Reply:
Like this in the new commit?